components/MarkdownHelpButton.js

```diff
@@ -0,0 +1,29 @@
+// Module imports
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+
+
+
+
+
+// Component imports
+import Button from './Button'
+
+
+
+
+
+const MarkdownHelpButton = ({ category, onClick }) => (
```

Comment:
See last PR for comment on this. We should move the editor help modal to here and let this component track state.